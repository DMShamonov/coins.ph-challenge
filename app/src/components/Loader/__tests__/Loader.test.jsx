import React from 'react';
import { shallow } from 'enzyme';

import Loader from '../index';

describe('`Loader` component should render correctly', () => {
  test('with default props', () => {
    expect(shallow(<Loader />)).toMatchSnapshot();
  });
});
