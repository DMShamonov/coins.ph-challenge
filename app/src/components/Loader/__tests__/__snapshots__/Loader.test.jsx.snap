// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`Loader\` component should render correctly with default props 1`] = `
<svg
  className="loader"
  viewBox="0 0 50 50"
>
  <circle
    className="path"
    cx="25"
    cy="25"
    fill="none"
    r="20"
    strokeWidth="5"
  />
</svg>
`;
